### DISCUSSION TITLE
Pengurusan Tabung

### DISCUSSION THREAD

**Discussion by rohaizanr** [Dev4w4n/e-masjid.my#137] [Ideas]
Modul Tabung tu, kalau Add Jumlah

Boleh pilih tarikh, pastu letak hari jumaat selepas AJK buat perkiraan total tabung masjid. Dengan buat penambahan Nilai Jumlah pada hari Jumaat.

Andaikata penggunaan duit tabung masjid, boleh add jumlah, lepas tu boleh pilih categori untuk guna duit tabung masjid atas tujuan apa.

Nanti dalam web boleh nampak jumlah transparent penggunaan tabung masjid dan akaun semasa tabung masjid. Access Modul hanya bendahari dan timbalan bendahari

Kategori di dalam Tabung boleh dimasukkan di dalam Module Dashboard backend dgn cara

Tambah Tabung
Guna Tabung

Dalam Tambah Tabung tersebut - maksudnya segala penambahan dalam kategori ini adalah berkaitan duit penambahan di dalam tabung dari segi jumlah pungutan semasa solat jumaat, hari raya, atau lain2.

Sama jgk dgn Guna Tabung maksudnya duit tersebut sekiranya di pilih, segala duit dari akaun semasa akaun ditolak dan diguna atas asbab sebab apa

Dari Main Web pulak, penggunaan tabung masjid boleh diletakkan supaya orang boleh tahu penggunaan duit masjid atas tujuan apa, dan juga ada 1 column dimana kita boleh klik Semasa posting mengenai aktiviti dijalankan. Column ni depend dgn masa penambahan atau penolakan duit tabung tersebut, maksudnya boleh klik select button untuk hide atau show.

Kalau show, maksudnya dalam posting aktiviti kita boleh pilih dana yang digunakan daripada pengeluaran duit tabung

**Comment by imie** [comment]
Ini contoh surau taman saya, mereka sekarang guna google docs sebab tiada sistem lagi.

Summary duit masuk dan duit keluar. 
![image](https://github.com/Dev4w4n/e-masjid.my/assets/2321338/4aa176c5-a108-4e21-9dcc-c35952e955df)

Mereka kutip duit sumbangan melalui akaun bank, jadi dalam module duit tabung tu, maybe boleh letak option dari bank berapa banyak untuk tarikh sekian sekian. AJK beli barang dan claim semula.

![image](https://github.com/Dev4w4n/e-masjid.my/assets/2321338/68b5713c-5170-47b4-8138-eddab1af0c45)

**Comment by rohaizanr** [comment]
terima kasih @imie 

pendapat saya, tabung ni macam sub-module untuk Pengurusan Kewangan Masjid 

contoh laporan surau ni, boleh park di sini https://github.com/Dev4w4n/e-masjid.my/discussions/125

**Reply by imie** [reply]
ok baik. terima kasih.
